documentation/views/portfolio.rst

```diff
@@ -57,33 +64,31 @@ Power profile measurements and forecasts
 The power profile shows total production and consumption over the selected time window.
 A switch allows the user to view the contribution of each asset type to either total as a stacked plot.
 Past time slots show measurement data, whereas future time slots show forecasts.
-When flexibility opportunities exist in the selected time window, the plot is overlaid with highlights (see :ref:`portfolio_flexibility_opportunities` ).
+When suggested changes exist in flexible schedules during the selected time window, the plot is overlaid with highlights (see :ref:`portfolio_flexibility_opportunities` ).
 
 
 .. _portfolio_flexibility_effects:
 
-Changes to the power profile due to flexibility actions
+Changes to the power profile due to flexible schedules
 =====================================================
 
-Just below the power profile, the net effect of flexibility actions that have previously been ordered is plotted.
-The profile indicates the change in power resulting from actions that are planned in the future, and from actions that had been planned in the past.
+A crucial goal of FlexMeasures is to visualise the opportunities within flexible schedules.
+This goal is not yet completely realised, but we show a mock here of how this could like when realised: 
+
+Just below the power profile, the net effect of flexible schedules that have previously been compute by FlexMeasures is plotted.
+The profile indicates the change in power resulting from schedules that are planned in the future, as well as from schedules that had been planned in the past.
 Positive values indicate an increase in production or a decrease in consumption, both of which result in an increased load on the network.
-For short-term flexibility actions, this is sometimes called up-regulation.
+For short-term flexibility changes, this is sometimes called up-regulation.
```

Comment:
Not a **change in flexibility**, but a change in power due to the **activation of flexibility**. Activating flexibility does not necessarily lead to a reduction in flexibility.